backend/sokdak/src/test/java/com/wooteco/sokdak/post/service/PostServiceTest.java

```diff
@@ -240,4 +240,24 @@ void deletePostWithComment() {
         Optional<Post> deletePost = postRepository.findById(post.getId());
         assertThat(deletePost).isEmpty();
     }
+
+    @DisplayName("해시태그가 있는 게시글 삭제 및 사용되지 않는 해시태그 삭제 기능")
```

Comment:
```suggestion
    @DisplayName("해시태그가 있는 게시글 삭제시 더 이상 사용되지 않는 해시태그 삭제 기능")
```
사용되지 않는 해시태그 삭제 기능을 테스트하기 위한 메서드이니 `및` 으로 두 기능을 테스트 하는 것으로 나타내는 것이 아니라 위와 같이 나타내면 어떨까요?

Reply:
수정했습니다!